tests/feast-wheel/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.poetry]
+name = "feast-wheel"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.10"
+feast = "^0.40.1"
```

Comment:
Can you test **only** the `dask`/`dask-expr` dependency?

It's also fine to leave the `feast` test.

Reply:
In fact, poetry2nix can build a project that depend on `dask` and `dask-expr` as long as no extra packages are specified for `dask`.

The infinite recursion happens only when another package, like `feast`, depends on `dask[dataframe]`, given that `dask-expr` depends on `dask`, and `dask[dataframe]` depends on `dask-expr`.